src/analysis/complex/basic.lean

```diff
@@ -93,6 +93,31 @@ by rw [norm_int, _root_.abs_of_nonneg]; exact int.cast_nonneg.2 hn
 @[continuity] lemma continuous_norm_sq : continuous norm_sq :=
 by simpa [← norm_sq_eq_abs] using continuous_abs.pow 2
 
+@[simp, norm_cast] lemma nnnorm_coe_real (r : ℝ) : ∥(r : ℂ)∥₊ = ∥r∥₊ :=
+subtype.ext $ by simp only [norm_real, coe_nnnorm]
```

Comment:
Does
```suggestion
subtype.ext norm_real
```
work? If so, this lemma should probably be called `nnorm_real` to match `norm_real`

Reply:
nearly, not quite. I also stripped some of the `cast`s from the others, but maybe that was hasty